src/documentation/about/features.malloynb

```diff
@@ -18,24 +18,24 @@ in any query at any level of granularity or dimensionality. Malloy retains enoug
 to perform this calculation no matter how you ask for it. Reusable Aggregates help improve accuracy.
 
 ### Reusable Dimensional calculations
-Dimensional (Scalar calculations) can also be introduced into the model. Dimensional calculation are useful
-mapping values, bucketing results and data cleanup.
+Dimensions (scalar calculations) can also be introduced into the model. Dimensional calculations are useful
+mapping values, bucketing results, and data cleanup.
 
 ### Maintains Relationships
 SQL's <code>SELECT</code> statement flattens the namespace into a wide table. Malloy retains the graph relationship
-of data lets you access and correctly perform computations and any place in the graph.
+of data and lets you access and correctly perform computations at any place in the graph.
 
 ### Reusable Queries
-Queries can be introduced into a Malloy model and accessed by name.  One benefit is that the
-queries are always accurate.  Think of a Malloy model as a data function library.
-Queries can also be used to create [nested subtables](../language/nesting.malloynb) in other queries.
+Queries can be introduced into a Malloy model and accessed by name. One benefit is that the
+queries are always accurate. Think of a Malloy model as a data function library.
```

Comment:
we have a comment from a user requesting we not say this, any data scientist would recognize this is as an impossible claim.